contracts/strategies/PlatypusVoter.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.3;
+
+import "../interfaces/IWAVAX.sol";
+import "../interfaces/IVePTP.sol";
+import "../lib/SafeERC20.sol";
+import "../lib/Ownable.sol";
+import "../lib/ERC20.sol";
+
+contract PlatypusVoter is Ownable, ERC20 {
+    using SafeERC20 for IERC20;
+    using SafeMath for uint256;
+
+    IWAVAX private constant WAVAX = IWAVAX(0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7);
+    address public constant PTP = address(0x22d4002028f537599bE9f666d1c4Fa138522f9c8);
+    IVePTP public constant vePTP = IVePTP(0x5857019c749147EEE22b1Fe63500F237F3c1B692);
+
+    address public immutable devAddr;
+    address public voterProxy;
+    bool public depositsEnabled;
```

Comment:
introduce toggle for this flag if we are using it here. We also have the capability to do this at the strategy level

Reply:
Added, forgot the setter!